### PR TITLE
import mtevbusted implementation

### PR DESCRIPTION
**Pull request by postwait**

### CODE REVIEW

File: test/Makefile.in [hunk 1]
```diff
@@ -64,20 +66,28 @@ test_harness:       $(TESTHARNESS_OBJS)
 	@echo "- linking $@"
 	$(Q)$(CC) -L../src $(LDFLAGS) $(UNWINDLIB) -o $@ $(TESTHARNESS_OBJS) $(LIBS) -lmtev
 
-busted:
-	@busted --version >/dev/null 2>/dev/null || \
-		(echo "Please instead busted: luarocks install busted" && false)
+mtevbusted-script:	mtevbusted-script.in
```
Comment (gallison93) [inline test/Makefile.in]: why not use autoconf for this? via AC_CONFIG_FILES

Comment (postwait) [inline test/Makefile.in]: Mainly so that one could override it.. but we could do that in autoconf as well.

**Comment by rileyberton** [comment]
Looks good to me

**Comment by gallison93** [comment]
lgtm

**Pull request merged**